packages/openactive-broker-microservice/test/util/item-transforms-test.js

```diff
@@ -0,0 +1,102 @@
+const { expect } = require('chai');
+const {invertFacilityUseItems, createItemFromSubEvent} = require('../../src/util/item-transforms');
+
+describe('test/utils/item-transforms-test', () => {
+  describe('invertFacilityUseItems', () => {
+    it('should return the input items when no invertibleFacilityUseItems are present', () => {
+      const items = [
+        { data: { individualFacilityUse: null } },
+        { data: { individualFacilityUse: [] } },
+      ];
+      const result = invertFacilityUseItems(items);
+      expect(result).to.equal(items);
+    });
+
+    it('should invert FacilityUse items and concatenate them with other items', () => {
+      // Test Objects
+      const facilityUseItems = [
+        { 
+          state: 'updated',
+          data: { 
+            '@context': 'https://openactive.io/',
+            name: 'Facility Use 1',
+            '@type': 'FacilityUse', 
+            individualFacilityUse: [{ '@type': 'IndividualFacilityUse', '@id': '1' }] 
+          } 
+        },
+        { 
+          state: 'updated',
+          data: { 
+            name: 'Facility Use 2',
+            '@context': 'https://openactive.io/',
+            '@type': 'FacilityUse', 
+            individualFacilityUse: [{ '@type': 'IndividualFacilityUse', '@id': '2' }] 
```

Comment:
suggestion: Have multiple IFUs in this one just to verify the flatmapping aspect